src/Helpers/VsCommands.cs

```diff
@@ -13,11 +12,14 @@ public static void Initialize()
         {
             CommandsAsString = ReadCommandsAsString();
             CheckEmptyEntries(CommandsAsString);
+            commands = CommandsAsString
+                .Split(new string[] { Environment.NewLine }, StringSplitOptions.RemoveEmptyEntries)
+                .ToImmutableArray();
         }
 
         public static string CommandsAsString { get; private set; }
 
-        private static ImmutableArray<string> commands;
+        private static ImmutableArray<string> commands = new ImmutableArray<string>();
```

Comment:
I just did the bare minimum to prevent the crash since I saw that the call to Initialize had been removed and didn't really think about the original implementation. 
That being said your way looks better, I can't remember why I used a separate Initialize method when I first built the control.

Reply:
So either 
```
private static ImmutableArray<string> commands = new ImmutableArray<string>();
```
or

```
 private ImmutableArray<string> Commands
         => commands == null || commands == ImmutableArray<string>.Empty
            ? commands = VsCommands.ParseCommands(commandsString)
            : commands;
```

Reply:
ImmutableArray is struct

Reply:
I don't wanna sound rude, but have you tested the code?
Without the initialization it fails with an Uninitialized-exception and with etiher solution above it does not.

Reply:
Without the initialization it crashes on search here:

```
var results = Commands.Where(c => c.IndexOf(searchText, StringComparison.InvariantCultureIgnoreCase) > -1);

```

So it is not null as you point out but still != ImmutableArray<string>.Empty and will therefore not be initialized..
Adding 
```
 private ImmutableArray<string> commands = ImmutableArray<string>.Empty;
```
would also work.
I don't wanna get into a big argument about this, I just want the control to work and not crash, since the CustomControls are what I use most for this extension.

Reply:
I pulled your branch and I like what you have done, the only thing missing would be the 
```
private ImmutableArray<string> commands = ImmutableArray<string>.Empty;
```
So if you would like to add that I will abandon this PR.